test/lib/builder/builder.js

```diff
@@ -1614,14 +1690,50 @@ const libraryJTree = {
 		"configuration": {
 			"paths": {
 				"src": "main/src"
-			}
+			},
+			"propertiesFileSourceEncoding": "ISO-8859-1"
```

Comment:
Why this change?

Reply:
Looks like I made copy&paste to the wrong object. Done.